action.yml

```diff
@@ -4,6 +4,12 @@ inputs:
   token:
     description: github token to clone / push with
     required: false
+  source:
+    description: The remote branch's commit_id when using `git push`.
+    required: false
+  origin:
+    description: The origin branch's commit_id when using `git push`.
+    required: false
```

Comment:
could you add something to README about this?

Reply:
@asottile I have added a section to README.

Reply:
no, you rewrote the readme.  don't do that

Reply:
I fixed typos. I can revert it if you wish.

Reply:
I am not a native speaker but a sentence should start with a capital letter.